resources/options.yaml.template

```diff
@@ -0,0 +1,11 @@
+options:
+  identityProvider: BASE_OC_IDP
+  hub:
+    baseDomain: BASE_DOMAIN
+    user: BASE_USER
+    password: BASE_PASSWORD
+    kubecontext: BASE_KUBECONTEXT
+  clusters:
+  - name: MANAGED_CLUSTER_NAME
+    kubeconfig: MANAGED_KUBECONFIG
+    basedomain: MANAGED_BASEDOMAIN
```

Comment:
CICD is now providing more fields in their `options.yaml`; therefore, we should remain consistent with the fields we have in our template file.

Reply:
Good, but let's delete the old template in the root directory.